gems/sorbet-runtime/lib/types/enum.rb

```diff
@@ -280,14 +280,18 @@ def _bind_name(const_name)
 
   sig {returns(T::Boolean)}
   def self.started_initializing?
-    @started_initializing = T.let(@started_initializing, T.nilable(T::Boolean))
-    @started_initializing ||= false
+    unless defined?(@started_initializing)
+      @started_initializing = T.let(false, T.nilable(T::Boolean))
+    end
+    T.must(@started_initializing)
```

Comment:
This is the cleanest way I could find to do the `T.let` while keeping the `defined?` check. I am open to alternatives that might look nicer.